helpdesk_mgmt/views/helpdesk_dashboard_view.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
There is a warning in travis
helpdesk_mgmt/views/helpdesk_dashboard_view.xml:1: [W7908(missing-newline-extrafiles), ]  Missing newline

Please, indent the code

Reply:
It's preferable to reduce diff instead of indenting.